content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md

```diff
@@ -64,6 +64,8 @@ spec:
       topologyKey: <string>
       whenUnsatisfiable: <string>
       labelSelector: <object>
+      nodeAffinityPolicy: <string> # optional; alpha since v1.25
```

Comment:
it's not an arbitrary string. I would say something like `[Honor|Ignore]`

Also, in line 51, it says `Here is an example`. But it's not an example. Maybe it should say, something like: `The usage of this field looks like the following:`